### DISCUSSION TITLE
GitHub Jobs are deprecated. Why "Available for hire" still in settings?

### DISCUSSION THREAD

**Discussion by yusufpapurcu** [community/community#9879] [Other Feature Feedback, Questions, & Ideas] [answered]
GitHub Jobs are deprecated. Why "Available for hire" still in settings?

**Comment by gyoshev** [comment, marked as answer]
Wanted to ask the same question and got here. There's also a [docs page](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-user-account/managing-user-account-settings/what-does-the-available-for-hire-checkbox-do) that shows a deprecation notice.

**Reply by Enough7** [reply]
One can still find the deprecation notice in the [Commit History of  github / docs](https://github.com/github/docs/commit/911bd330835f80988a37129febb582143deaa34d#diff-84af8a228d348cdbe22603a00bb5bcb0cdcfbc5f7c7f3dd8e8da8f146e73b5feL18-L22)
